explorer/src/components/TokenStatsCard.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { LoadingCard } from "./common/LoadingCard";
+import { ErrorCard } from "./common/ErrorCard";
+import {
+  useCoinGeckoCategoryStats,
+  COIN_GECKO_SOLANA_CATEGORY,
+} from "utils/coingecko";
+
+import { formatDollarValue } from "utils";
+
+import { TableCardBody } from "./common/TableCardBody";
+
+export function TokenStatsCard() {
+  const [error, loading, categoryStats] = useCoinGeckoCategoryStats(
+    COIN_GECKO_SOLANA_CATEGORY
+  );
+
+  if (error) return <ErrorCard text={error.toString()} />;
+
+  if (loading) return <LoadingCard />;
+
+  if (categoryStats)
+    return (
+      <div className="card">
```

Comment:
Can you replace layout here with cards like on the front page: 
![image](https://user-images.githubusercontent.com/264380/120199988-733f4880-c1e9-11eb-9921-bbf08d20b161.png)

Reply:
Sure!